src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -1323,20 +1327,50 @@ class FrontendGenImpl {
     int nOut = 0;
     getNodeInputs(node, inputs);
     nOut = node.output().size();
+    std::vector<Type> givenOutputTypes;
+
+    // We lack a way of specifying import behavior for custom domains. For now
+    // some are hard-coded here, but an extension specification would be
+    // preferred.
+    if (node.domain().compare("com.microsoft") == 0) {
+      Type outElementType = {};
+      if (opName == "DequantizeLinear") {
+        outElementType =
+            inputs.at(1).getType().cast<ShapedType>().getElementType();
```

Comment:
Could we use the new cast style, pls?
```suggestion
            cast<ShapedType>(inputs.at(1).getType()).getElementType();
```

Reply:
I'll take this in follow-up.